### PR TITLE
MULE-13034: Escape all error handlers.

### PR DESCRIPTION
**Pull request by afelisatti**

### CODE REVIEW

File: modules/http/src/test/java/org/mule/module/http/internal/listener/ErrorRequestHandlerTestCase.java [hunk 1]
```diff
@@ -26,23 +27,42 @@
...
-public class NoListenerRequestHandlerTestCase extends AbstractMuleTestCase
+@RunWith(Parameterized.class)
+public class ErrorRequestHandlerTestCase extends AbstractMuleTestCase
 {
     private final static String TEST_REQUEST_INVALID_URI  = "http://localhost:8081/<script>alert('hello');</script>";
     private final HttpRequestContext context = mock(HttpRequestContext.class, RETURNS_DEEP_STUBS);
     private final HttpResponseReadyCallback responseReadyCallback = mock(HttpResponseReadyCallback.class);
-    private NoListenerRequestHandler noListenerRequestHandler;
+
+    @Parameterized.Parameter(0)
```
Comment (facundovs) [inline modules/http/src/test/java/org/mule/module/http/internal/listener/ErrorRequestHandlerTestCase.java]: static import.

File: modules/http/src/test/java/org/mule/module/http/internal/listener/ErrorRequestHandlerTestCase.java [hunk 2]
```diff
@@ -26,23 +27,42 @@
...
+    public ErrorRequestHandler errorRequestHandler;
+
+    @Parameterized.Parameter(1)
+    public String format;
+
+    @Parameterized.Parameters(name = "{0}")
+    public static Collection<Object[]> parameters()
+    {
+        return Arrays.asList(new Object[][] {
```
Comment (facundovs) [inline modules/http/src/test/java/org/mule/module/http/internal/listener/ErrorRequestHandlerTestCase.java]: static import

Comment (facundovs) [reply modules/http/src/test/java/org/mule/module/http/internal/listener/ErrorRequestHandlerTestCase.java]: is the code style right here?

Comment (afelisatti) [reply modules/http/src/test/java/org/mule/module/http/internal/listener/ErrorRequestHandlerTestCase.java]: Yes, cause it's an array definition.

**Pull request merged**